Microsoft.Toolkit.Uwp.SampleApp/SamplePages/PullToRefreshListView/PullToRefreshListViewPage.xaml.cs

```diff
@@ -40,5 +42,17 @@ private void ListView_RefreshCommand(object sender, EventArgs e)
         {
             AddItems();
         }
+
+        private void ListView_RefreshIntentCanceled(object sender, EventArgs e)
+        {
+
+        }
+
+        private DelegateCommand RefreshIntentCanceled => _refreshIntentCanceledCommand ?? (_refreshIntentCanceledCommand = new DelegateCommand(
+            () =>
+            {
+                _items.Insert(0, new Item { Title = $"Item {_items.Count} canceled" });
```

Comment:
It feels weird that the sample is adding an item when intent is canceled. It might work better if it's a mesagedialog or even just a NOP.